backend/sXb-service/Repos/Interfaces/IBookApi.cs

```diff
@@ -1,14 +1,12 @@
-using sXb_service.Models;
-using sXb_service.Repos.Base;
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-
-namespace sXb_service.Repos.Interfaces
-{
-    public interface IAuthorRepo : IRepo<Author>
-    {
-
-    }
-}
+﻿using sXb_service.Helpers;
+using sXb_service.Models;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace sXb_service.Repos.Interfaces
+{
+    public interface IBookApi
+    {
+        Task<Paging<BookApiResult>> FindBook(string term, int page);

```

Comment:
This interface Should be located underneath services, beside the implementations.